tests/test_coverage.py

```diff
@@ -109,53 +112,6 @@ def create_identifier_mapping(self, batch):
         eq_("Here's your title!", edition.title)
         eq_(id1, edition.primary_identifier)
 
-    def test_finalize_license_pool(self):
```

Comment:
It's been a while, but I think the code tested here has either been removed (the presentation_ready_on_success stuff) or it's tested in the new tests below.